src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -23,11 +46,16 @@ const ParentFamilyHome = () => {
         </div>
         <div className="between-profile-students">
           <h1 className="students-section-title">Students</h1>
-          <button className="family-page-button">Create New Student</button>
+          <button
+            className="family-page-button"
+            onClick={() => setModal(!modal)}
+          >
+            Create New Student
+          </button>
         </div>
         <div className="student-card-container">
           <h1 className="student-card-name">
-            <strong>Student Name: _____</strong>
+            <strong>Student Username: ____</strong>
```

Comment:
Not sure how this actually looks on the website, but could you explain why you chose to put the ___ after the labels? This may be something to consider removing in the future.